urlchecker.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+# url testing
+# source urlchecker.sh
```

Comment:
Can you improve this documentation a bit?

Reply:
ah - you shouldn't be sourcing this file

Instead, use:

```
chmod ug+x urlchecker.sh
```

to mark it as executable. Then you can run it with `./urlchecker.sh`

Reply:
I'm having trouble getting this to work without source. I believe it is because when I try to update the arrays on line 27 and 30 they are not updated globally so the arrays just stay empty. I have tried to update line 31 to use done in a few different ways to save the resulting arrays but haven't had any luck. Any idea how I could do this?

Reply:
(resolved offline)